From 7ba8ea2cf06230e647b096f40d3006abec45f801 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Mon, 12 Jul 2021 16:33:20 +0200
Subject: [PATCH] include: missing sctp_chunk.h in Makefile.am

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1979334
Upstream Status: nftables commit 117ceb4f52711

commit 117ceb4f527119a6d44bf5e23f2ff7a8d116658a
Author: Pablo Neira Ayuso <pablo@netfilter.org>
Date:   Tue May 25 14:04:36 2021 +0200

    include: missing sctp_chunk.h in Makefile.am

    Fix make distcheck.

    Fixes: 0e3871cfd9a1 ("exthdr: Implement SCTP Chunk matching")
    Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
---
 include/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/Makefile.am b/include/Makefile.am
index 04a4a61..4ee5124 100644
--- a/include/Makefile.am
+++ b/include/Makefile.am
@@ -30,6 +30,7 @@ noinst_HEADERS = 	cli.h		\
 			osf.h		\
 			parser.h	\
 			proto.h		\
+			sctp_chunk.h	\
 			socket.h	\
 			rule.h		\
 			rt.h		\
-- 
2.31.1

